### PR TITLE
syntax fixes to swift 3 beta version

### PR DESCRIPTION
**Pull request by sairamkotha**

### CODE REVIEW

File: NVActivityIndicatorView/NVActivityIndicatorView.swift [hunk 1]
```diff
@@ -312,28 +312,25 @@ public enum NVActivityIndicatorType: Int {
...
     
     /// Default color. Default value is UIColor.whiteColor().
-    public static var DEFAULT_COLOR = UIColor.whiteColor()
+    public static var DEFAULT_COLOR = UIColor.white()
     
     /// Default padding. Default value is 0.
     public static var DEFAULT_PADDING: CGFloat = 0
     
-    /// Default size of activity indicator view in UI blocker. Default value is 60x60.
-    public static var DEFAULT_BLOCKER_SIZE = CGSizeMake(60, 60)
```
Comment (ninjaprox) [inline NVActivityIndicatorView/NVActivityIndicatorView.swift:324]: Please keep this variable as it is used to set globally.

File: NVActivityIndicatorView/NVActivityIndicatorViewable.swift [hunk 2]
```diff
@@ -24,39 +24,36 @@ public extension NVActivityIndicatorViewable where Self: UIViewController {
...
      - parameter padding: view's padding. Default padding is 0.
      */
-    public func startActivityAnimating(size: CGSize? = nil, message: String? = nil, type: NVActivityIndicatorType? = nil, color: UIColor? = nil, padding: CGFloat? = nil) {
+    public func startActivityAnimating(_ size: CGSize = CGSize(width: 60, height: 60), message: String? = nil, type: NVActivityIndicatorType? = nil, color: UIColor? = nil, padding: CGFloat? = nil) {
         let activityContainer: UIView = UIView(frame: view.bounds)
-        
         activityContainer.backgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.5)
         activityContainer.restorationIdentifier = activityRestorationIdentifier
         
-        let actualSize = size ?? NVActivityIndicatorView.DEFAULT_BLOCKER_SIZE
```
Comment (ninjaprox) [inline NVActivityIndicatorView/NVActivityIndicatorViewable.swift:39]: Should use `DEFAULT_BLOCK_SIZE` as mentioned above.

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift [hunk 3]
```diff
@@ -10,10 +10,10 @@ import UIKit
 
 class NVActivityIndicatorAnimationAudioEqualizer: NVActivityIndicatorAnimationDelegate {
     
-    func setUpAnimationInLayer(layer: CALayer, size: CGSize, color: UIColor) {
+    func setUpAnimationInLayer(_ layer: CALayer, size: CGSize, color: UIColor) {
```
Comment (ninjaprox) [inline NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift]: As API design guidelines, the method name should be `setUpAnimation(in layer: CALayer, size: CGSize, color: UIColor)`. What do you think?

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift [hunk 4]
```diff
@@ -10,10 +10,10 @@ import UIKit
...
         let lineSize = size.width / 9
-        let x = (layer.bounds.size.width - lineSize * 7) / 2
-        let y = (layer.bounds.size.height - size.height) / 2
+        let x = (layer.bounds.width - lineSize*7)/2
+        let y = (layer.bounds.height - size.height)/2
```
Comment (ninjaprox) [inline NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift]: Please keep convention consistently. It should be whitespace between operand and operator.

**Comment by ninjaprox** [comment]
In general, there are some methods needed to rename. What do you think?

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift (hunk 4 above)
Comment (sairamkotha) [inline NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift]: Fine I will update.

File: NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift (hunk 3 above)
Comment (sairamkotha) [inline NVActivityIndicatorView/Animations/NVActivityIndicatorAnimationAudioEqualizer.swift]: Ya I will update syntax 👍

**Comment by sairamkotha** [comment]
@ninjaprox  Now check

**Comment by ninjaprox** [comment]
Nicely done 👍. Additionally, please also check out comments on `DEFAULT_BLOCKER_SIZE` above. Thanks

**Comment by sairamkotha** [comment]
@ninjaprox fixed default blocker size

File: NVActivityIndicatorView/NVActivityIndicatorViewable.swift [hunk 5]
```diff
@@ -30,13 +30,14 @@ public extension NVActivityIndicatorViewable where Self: UIViewController {
      - parameter color: color of activity indicator view. Default color is white.
      - parameter padding: view's padding. Default padding is 0.
      */
-    public func startActivityAnimating(_ size: CGSize = CGSize(width: 60, height: 60), message: String? = nil, type: NVActivityIndicatorType? = nil, color: UIColor? = nil, padding: CGFloat? = nil) {
+    public func startActivityAnimating(_ size: CGSize?, message: String? = nil, type: NVActivityIndicatorType? = nil, color: UIColor? = nil, padding: CGFloat? = nil) {
```
Comment (ninjaprox) [inline NVActivityIndicatorView/NVActivityIndicatorViewable.swift]: `size` param should have default value `nil`.

Comment (sairamkotha) [inline NVActivityIndicatorView/NVActivityIndicatorViewable.swift]: Done @ninjaprox

**Comment by sairamkotha** [comment]
@ninjaprox made default size value to nil

**Pull request merged**